src/annotator/config/index.js

```diff
@@ -90,12 +90,12 @@ const configDefinitions = {
   appType: {
     allowInBrowserExt: true,
     defaultValue: null,
-    getValue: settings => settings.hostPageSetting('appType'),
+    getValue: (settings, name) => settings.hostPageSetting(name),
```

Comment:
The `(settings, name) => settings.hostPageSetting(name)` closure is duplicated several times. Those could be combined into a single closure, or alternatively you could make `getValue` optional and default to calling `settings.hostPageSetting(name)` if no `getValue` function is provided.

Reply:
I'd rather not get too cute with implicit options so I'll pass a function to de-duplicate those calls but still make getValue requried.